spec/validators/well_validator_spec.rb

```diff
@@ -9,22 +9,34 @@
     context 'invalid' do
       it 'returns an error when there are multiple libraries but no tags' do
         # Going through the library_ids= method to create the aliquots
-        well.library_ids = create_list(:pacbio_library, 2, :untagged).collect(&:id)
+        library1 = create(:pacbio_library, :untagged)
+        library2 = create(:pacbio_library, :untagged)
+
+        well.used_aliquots_attributes = [{ source_id: library1.id, source_type: 'Pacbio::Library', volume: 10, concentration: 20, aliquot_type: :derived, template_prep_kit_box_barcode: '033000000000000000000' },
```

Comment:
If you want to for these tests you could do `well.used_aliquots = build_list(:aliquot )...` instead of assigning to `used_aliquots_attributes`. The reason we used pool_ids and library_ids before was because it kept aliquots and libraries/pools in sync but that is no longer an issue.